tests/testHtmIndex.py

```diff
@@ -239,6 +242,16 @@ def testIngest(self):
         cent = make_coord(*tupl)
         cat = loader.loadSkyCircle(cent, self.search_radius, filterName='a')
         self.assertTrue(len(cat) > 0)
+        # test that a catalog can be loaded even with a name not used for ingestion
+        butler = dafPersist.Butler(self.test_repo_path)
+        config = LoadIndexedReferenceObjectsConfig()
+        config.ref_dataset_name = "diff_ref_name"
```

Comment:
This is a magic string used in multiple places.  Can you consolidate them into a single variable?